webapp/migrations/Version20230508135106.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230508135106 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Replace jobid with foreign key constraint.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('DROP INDEX jobid ON queuetask');
+        $this->addSql('ALTER TABLE queuetask ADD judgingid INT UNSIGNED DEFAULT NULL COMMENT \'Judging ID\', DROP jobid');
```

Comment:
It would be better to make this a `CHANGE COLUMN` statement (also in the down), so that data is preserved.